added configuration file analysis blog

SankBad opened the pull request:
This PR will add entry in toc.yaml and content-sources.yaml files for configuration file analysis blog. 
We used remote repo([src](https://github.com/aicoe-aiops/configuration-files-analysis)) for adding this blog. 
I tested the site locally and it looks as expected.

sesheta commented:
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign  after the PR has been reviewed.
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=operate-first%2Foperate-first.github.io).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/operate-first/operate-first.github.io/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
<!-- META={"approvers":[]} -->

oindrillac reviewed (changes requested):
@SankBad I get an error from the ssh url in the content-sources.yaml (when I try to build without manually cloning the repo). Can you replace it with https://github.com/aicoe-aiops/configuration-files-analysis.git?

sesheta commented:
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=operate-first%2Foperate-first.github.io).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/operate-first/operate-first.github.io/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
<!-- META={"approvers":[]} -->

sesheta commented:
@SankBad: PR needs rebase.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

SankBad commented:
@oindrillac , Thanks for pointing out.
I changed ssh key to a web url.

oindrillac commented:
@SankBad I think the ignore key added in #48 for the continuous-deployment repo got moved. can you please move it back?

SankBad commented:
Hi @oindrillac , 

ignore key is available in this PR. 
I rebase my repo before making the changes.

cfchase commented on content-sources.yaml:
```diff
@@ -14,5 +14,10 @@
+- name: configuration file analysis
+  gitSrc: https://github.com/aicoe-aiops/configuration-files-analysis.git
+  dir: configuration-files-analysis/docs/blog
+  urlPrefix: config_analysis
```
I would stick with dash `config-analysis` instead of `config_analysis` in urls just to be consistent

SankBad commented:
@cfchase @oindrillac 
Thanks for the feedback. 
I fixed the format of urlPrefix and changed the location of ignore key.
Also, squashed the commits.

cfchase commented on content-sources.yaml:
```diff
@@ -16,3 +16,9 @@
+  gitSrc: https://github.com/aicoe-aiops/configuration-files-analysis.git
+  dir: configuration-files-analysis/docs/blog
+  urlPrefix: config-analysis
+
```
add `configuration-files-analysis`  in the Remote Content section in the [`.gitignore`](https://github.com/operate-first/operate-first.github.io/blob/master/.gitignore#L2)

sesheta commented:
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign  after the PR has been reviewed.
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=operate-first%2Foperate-first.github.io).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/operate-first/operate-first.github.io/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
<!-- META={"approvers":[]} -->

SankBad commented:
Hi @cfchase ,

Added entry in the [`.gitignore`](https://github.com/operate-first/operate-first.github.io/blob/master/.gitignore#L2) file

sesheta commented:
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/operate-first/operate-first.github.io/pull/47#pullrequestreview-514973876" title="Approved">cfchase</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=operate-first%2Foperate-first.github.io).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/operate-first/operate-first.github.io/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
<!-- META={"approvers":[]} -->

sesheta commented:
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/operate-first/operate-first.github.io/pull/47#pullrequestreview-514973876" title="Approved">cfchase</a>*, *<a href="https://github.com/operate-first/operate-first.github.io/pull/47#pullrequestreview-514978990" title="Approved">oindrillac</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign  after the PR has been reviewed.
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=operate-first%2Foperate-first.github.io).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/operate-first/operate-first.github.io/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
<!-- META={"approvers":[]} -->
